src/py/flwr/client/app.py

```diff
@@ -141,6 +147,21 @@ def _parse_args_run_client_app() -> argparse.ArgumentParser:
         default="0.0.0.0:9092",
         help="Server address",
     )
+    parser.add_argument(
+        "--retry_max_tries",
+        type=int,
+        default=1,
+        help="The maximum number of times the client will try to reconnect to the"
+        "server before giving up in case of a connection error. If set to None,"
+        "there is no limit to the number of tries.",
+    )
+    parser.add_argument(
+        "--retry_max_time",
+        type=float,
+        help="The maximum total amount of time before the client stops trying to"
+        "reconnect to the server before giving up in case of connection error."
+        "If set to None, there is no limit to the total time.",
```

Comment:
Same as before. Should we use `reconnect`, or `connect` (or other words)? Because this implies that the timer for retry_max_time starts when the client first encounters a connection error and begins attempting to reconnect, not from the initial connection attempt to the server.